frontend/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "@shopgate-project/product-list-add-to-cart",
+  "version": "1.0.0-rc.2",
+  "description": "Allows users to add desired quantity from a product list view.",
+  "license": "Apache-2.0",
+  "scripts": {
+    "lint": "eslint --parser babel-eslint --ignore-path ../.gitignore --ignore-path .eslintignore --ext .js --ext .jsx .",
+    "test": "jest",
+    "test:watch": "npm run test -- --watch",
+    "coverage": "jest --coverage",
+    "coverage:watch": "npm run coverage -- --watch"
+  },
+  "devDependencies": {
+    "@shopgate/eslint-config": "^6.4.1",
+    "@shopgate/pwa-common": "^6.4.1",
+    "@shopgate/pwa-common-commerce": "^6.4.1",
+    "@shopgate/pwa-core": "^6.4.1",
+    "jest": "^22.4.2",
+    "@shopgate/pwa-unit-test": "^6.4.1",
+    "@shopgate/pwa-ui-ios": "^6.4.1",
+    "@shopgate/pwa-ui-shared": "^6.4.1",
+    "babel-core": "^6.26.0",
+    "babel-plugin-react-transform": "^3.0.0",
+    "babel-plugin-transform-class-properties": "^6.24.1",
+    "babel-plugin-transform-export-extensions": "^6.22.0",
+    "babel-plugin-transform-object-rest-spread": "^6.26.0",
+    "babel-plugin-transform-react-inline-elements": "^6.22.0",
+    "babel-plugin-transform-react-jsx": "^6.24.1",
+    "babel-preset-env": "^1.6.1",
+    "babel-preset-react": "^6.24.1",
+    "eslint": "^4.19.1",
+    "glamor": "^2.20.40",
+    "prop-types": "^15.6.0",
+    "react": "^16.2.0",
+    "react-dom": "^16.2.0",
+    "react-helmet": "^5.2.0",
+    "react-transition-group": "^2.2.1",
+    "react-hot-loader": "^3.1.3",
+    "react-transform-catch-errors": "^1.0.2"
+  },
+  "dependencies": {
+    "@shopgate-ps/pwa-extension-kit": "^0.6.0"
```

Comment:
I don't think you are using the pwa-extension-kit so this can be removed -- I think

Reply:
Removed